client/config/login.js

```diff
@@ -0,0 +1,65 @@
+      // When the form is submitted, we validate there's an email and password entered
```

Comment:
@dropcat13 I believe since this file renders content we'd want it in the components directory, correct?